### DISCUSSION TITLE
help with mouse() JavaScript

### DISCUSSION THREAD

**Discussion by 00Her0** [nesbox/TIC-80#1350] [Q&A]
I 'm trying to use mouse() with JavaScript. I saw a code snippet in lua like this:
mx,my,md=mouse() --get x,y and pressed
I've tried stuff like that with js but nothing seemed to work, on variable held all the values separated by commas. How can I get them individually?

**Comment by RobLoach** [comment]
It returns an array...

``` js
var mouseOutput = mouse()
var x = mouseOutput[0]
var y = mouseOutput[1]
// ...
```

**Reply by 00Her0** [reply]
Oh, that makes much more sense now. Thank you!
